jobsdb/jobsdb.go

```diff
@@ -2761,11 +2763,9 @@ func (jd *HandleT) backupDSLoop(ctx context.Context) {
 		var opID int64
 		if isBackupConfigured() {
 			opID = jd.JournalMarkStart(backupDSOperation, opPayload)
-			success := jd.backupDS(backupDSRange)
-			if !success {
-				jd.removeTableJSONDumps()
-				jd.JournalMarkDone(opID)
-				continue
+			err := jd.backupDS(ctx, backupDSRange)
+			if err != nil {
+				jd.logger.Errorf("[JobsDB] :: Failed to backup jobs table %v. Err: %v", backupDSRange.ds.JobStatusTable, err)
```

Comment:
Don't we need this? 

jd.removeTableJSONDumps()

Reply:
since, in `backupTable` function we are already doing `os.Remove()`. So, I don't think we need to call `jd.removeTableJSONDumps()`.

Reply:
Got it.